README.md

```diff
@@ -10,6 +13,104 @@ The (evolving) protocol spec is rendered at [protocol.penumbra.zone][protocol].
 
 The (evolving) API documentation is rendered at [rustdoc.penumbra.zone][rustdoc].
 
+To participate in our test network, [keep reading below](#getting-started-on-the-test-network).
+
+For instructions on how to set up a node, [jump down and read on](#running-a-penumbra-node).
+
+## Getting started on the test network
+
+[Penumbra Labs][Penumbra] runs a test network for the latest version of the Penumbra protocol, and
+we would be delighted for you to try it out! Keep in mind that especially in the early stages of
+development, we expect many things to break. If you encounter bugs or wish for features to exist,
+we'd love for you to reach out to us on our [Discord server][Discord].
+
+### Building `pcli`
+
+To get started with the Penumbra test network, you will first need to download and build the
+Penumbra command line light wallet, `pcli`. This requires that you install a recent stable version
+of the Rust compiler, installation instructions for which you can find
+[here](https://www.rust-lang.org/learn/get-started).
+
+Once you have installed Rust and its build tool `cargo`, you can build the `pcli` light wallet by
+cloning this repository (`git clone https://github.com/penumbra-zone/penumbra`) and running from the
+newly cloned project:
+
+```bash
+cargo build --release --bin pcli
+```
+
+### Generating your wallet
+
+When working with `pcli`, the level of diagnostic information printed is dependent on the `RUST_LOG`
+environment variable. To see progress updates and other logged information while running `pcli`, we
+recommend you set `export RUST_LOG=info` in your terminal.
+
+On first installation of `pcli`, you will need to generate a fresh wallet to use with Penumbra. You
+should see something like this:
+
+```bash
+$ cargo run --quiet --release --bin pcli wallet generate
+Saving wallet to /home/$USER/.local/share/pcli/penumbra_wallet.json
+Saving backup wallet to /home/$USER/.local/share/testnet-archive/penumbra-valetudo/.../penumbra_wallet.json
```

Comment:
hmm, it's probably worth changing the name to include `penumbra` as you pointed out, since Linux is discarding the prefixes.

Reply:
Done.